test/setup.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2017 Super Adventure Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/nafundi/super-adventure/blob/master/NOTICE.
+
+This file is part of Super Adventure. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of Super Adventure,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+
+// TODO: Without this, PhantomJS throws an error indicating that Symbol is not
+// defined. Is Babel not defining Symbol? Should we switch to Headless Chrome
+// over PhantomJS?
```

Comment:
I've created an issue for this: https://github.com/nafundi/super-adventure/issues/35. Any thoughts are much appreciated. I figure we can tackle this question in a future PR.